openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -134,7 +134,7 @@ <h5 class="message__title">{% trans "Your Feedback Has Been Submitted" %}</h5>
                                     <ol class="list list--fields submission__feedback__fields {{ has_submitted_feedback|yesno:"is--hidden," }}"
                                             {{ has_submitted_feedback|yesno:'aria-hidden=true,aria-hidden=false' }}>
                                         <li class="field field-group field--radio feedback__overall">
-                                            <h4 id="feedback__overall__prompt__{{ xblock_id }}" class="field-group__label">{% trans "Select the statements below that best reflect your experience with peer assessments" %}:</h4>
+                                            <h5 id="feedback__overall__prompt__{{ xblock_id }}" class="field-group__label">{% trans "Select the statements below that best reflect your experience with peer assessments" %}:</h5>
```

Comment:
@staubina Is this the same level of heading from which we've been removing colons? If it requires punctuation we can use a period.